keyboards/southpole/keymaps/foobeard/keymap.c

```diff
@@ -0,0 +1,78 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "southpole.h"
+
+enum layers {
+  _QWERTY,
+  _RAISE
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+#define ______ KC_TRNS
```

Comment:
QMK defines a keycode for this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

Please delete Line 27, and replace all instances of  
`______` (six underscores) with  
`_______` (seven underscores).